packages/publisher/src/generate-packages.ts

```diff
@@ -123,83 +111,43 @@ async function outputFilePath(pkg: AnyPackage, filename: string): Promise<string
   return full;
 }
 
-interface Dependencies {
-  [name: string]: string;
-}
-
-export function createPackageJSON(typing: TypingsData, version: string, packages: AllPackages): string {
+export function createPackageJSON(typing: TypingsData, version: string): string {
   // Use the ordering of fields from https://docs.npmjs.com/files/package.json
   const out: {} = {
-    name: typing.fullNpmName,
+    name: typing.name,
     version,
     description: `TypeScript definitions for ${typing.libraryName}`,
     // keywords,
-    homepage: `https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/${typing.name}`,
+    homepage: `https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/${typing.typesDirectoryName}`,
```

Comment:
In the previous comment we were adding a version to typesDirectoryName but now we're not; definitely should double check that the paths are what we expect.

Reply:
I guess that's true, yeah; obviously one can move v18 to a v18 dir and the paths shouldn't break or differ.